source/jormungandr/tests/journey_common_tests.py

```diff
@@ -79,6 +79,17 @@ def test_journeys(self):
 
         self.check_context(response)
 
+    def test_address_in_stop_points_on_journeys(self):
+        response = self.query_region(journey_basic_query)
+        for j in response['journeys']:
+            for s in j['sections']:
+                if 'stop_point' in s['from']:
+                    address = get_not_null(s['from']['stop_point'], 'address')
+                    is_valid_address(address, depth_check=0)
+                if 'stop_point' in s['to']:
+                    address = get_not_null(s['to']['stop_point'], 'address')
+                    is_valid_address(address, depth_check=0)
```

Comment:
maybe assert somewhere that there is at least one section with `'stop_point' in s['from']` and one with `'stop_point' in s['to']`, otherwise you are not testing anything